libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.EditActions.schema

```diff
@@ -11,8 +11,8 @@
     "properties": {
         "changeType": {
             "type": "string",
-            "title": "Change Type",
-            "description": "The change type to apply to current dialog",
+            "title": "Type of change",
+            "description": "Type of change to apply to the current conversation flow.",
```

Comment:
>conversation flow [](start = 67, length = 17)

action plan.

Reply:
The visual canvas shows a flow - it is a flow-chart. We're trying to use commonly understood words to explain the concepts.